watcher/tool/watcher/rollups.go

```diff
@@ -76,9 +73,22 @@ func receiveAndCreateRollups(ctx context.Context, client *influxdb.Client, ch <-
 				}
 
 				for _, rollup := range rollups {
-					err := client.CreateRollup(rollup)
-					if err != nil {
-						log.Errorf("failed to create rollup %v: %v", rollup, trace.DebugReport(err))
+					switch update.EventType {
+					case watch.Added:
+						err := client.CreateRollup(rollup)
+						if err != nil {
+							log.Errorf("failed to create rollup %v: %v", rollup, trace.DebugReport(err))
```

Comment:
I'm wondering if you ever need to retry the operation (i.e. `CreateRollup`) if the API has failed? How does the caller (i.e. one updating the configuration map) know whether the changes were applied?